### PR TITLE
Add Darryl Llanura details

### PR DESCRIPTION
**Pull request by Ghost-017**

### CODE REVIEW

File: assets/data/devs.json [hunk 1]
```diff
@@ -338,5 +338,13 @@
...
         "about": "Loves learning and baking bread 🍞",
         "username": "jose-bamboo"
+    },
+    {
+        "name": "Darryl Llanura",
+        "roles": [
+            "Flutter Developer"
+        ],
+        "about": "Enjoys making apps with Flutter!",
+        "username": "Ghost-017"
```
Comment (Jansalvador1445) [inline assets/data/devs.json:348]: Can you add `socials` on your data? Thanks

```
socials: {
"facebook" : "",
"twitter" : "",
"linkedin" : ""
}

**Comment by Jansalvador1445** [comment]
and Please include issue number here.

File: assets/data/devs.json (hunk 1 above)
Comment (Ghost-017) [reply assets/data/devs.json:348]: yeah, sure!

**Comment by Ghost-017** [comment]
> and Please include issue number here.

it's #230

**Comment by Jansalvador1445** [comment]
Has `Conflicts` can you rebase it again? (I just merged  something on master).

**Comment by Jansalvador1445** [comment]
Still has `conflict` and then don't delete `pubspec.lock`

**Comment by Jansalvador1445** [comment]
Siguro, Close this PR then recreate one. 

`git checkout master`
`git pull`
`git checkout -b branch_name`
`//Add Codes or changes`
`git add .`
`git commit -m "message here"`
`git push --set-upstream origin <branch_name>`

**Pull request closed without merging**